src/iotHubMessageExplorer.ts

```diff
@@ -11,6 +11,54 @@ import { IoTHubMessageBaseExplorer } from "./iotHubMessageBaseExplorer";
 import { DeviceItem } from "./Model/DeviceItem";
 import { TelemetryClient } from "./telemetryClient";
 import { Utility } from "./utility";
+import { Client } from 'azure-iot-device';
+import { ConnectionString } from 'azure-iot-common';
+import { Send } from "express-serve-static-core";
+
+export class SendStatus {
+    private id: string;
+    private succeed: number;
+    private failed: number;
+    private total: number;
+
+    constructor(id: string, total: number) {
+        this.id = id;
+        this.succeed = 0;
+        this.failed = 0;
+        this.total = total;
+    }
+    
+    public getId(): string {
+        return this.id;
+    }
+    public getSucceed(): number {
+        return this.succeed;
+    }
+    public getFailed(): number {
+        return this.failed;
+    }
+    public getTotal(): number {
+        return this.total;
+    }
+
+    private AddSucceed(): void {
+        this.succeed++;
+    }
+    private AddFailed(): void {
+        this.failed++;
+    }
+    public newStatus(succeed: boolean) {

```

Comment:
seems no need to wrap addSucceed/addFailed into one method. It's more clear to me to just addSucceed or addFailed.

Reply:
Modified.